packages/core/bootstrap/test/unit/validator.test.ts

```diff
@@ -7,15 +7,6 @@ describe('Validator', () => {
       endpoint: false,
     }
 
-    it('errors if no input data is supplied', () => {
```

Comment:
We still want this right? Seems like a useful test. Might just have to try catch it.